app/code/Magento/InventoryApi/Test/_files/source_items.php

```diff
@@ -18,15 +18,29 @@
 $sourceItemsSave = Bootstrap::getObjectManager()->get(SourceItemsSaveInterface::class);
 
 /**
+ * SKU-1 - Default Source(id:1) - 7.7qty
  * SKU-1 - EU-source-1(id:10) - 5.5qty
  * SKU-1 - EU-source-2(id:20) - 3qty
  * SKU-1 - EU-source-3(id:30) - 10qty (out of stock)
  * SKU-1 - EU-source-4(id:40) - 10qty (disabled source)
  *
+ * SKU-2 - Default Source(id:1) - 5.5qty
  * SKU-2 - US-source-1(id:30) - 5qty
  */
 $sourcesItemsData = [
     [
+        SourceItemInterface::SOURCE_ID => 1, // Default source
```

Comment:
Need to replace on DefaultSourceProviderInterface

Reply:
do you mean by defining a const in `DefaultSourceProviderInterface` interface or calling a method on an instance retrieved by the Object Manager?

Reply:
We shouldn't use Default Source because as I mentioned in another tasks Inventory module doesn't have dependency on `InventoryCatalog`
Tests have to be green without `InventoryCatalog`
So we need to create one more stock if it is needed